File: WebShopManagement/Webshop/Webshop/DataContracts/CreateCustomerModel.cs
namespace Webshop.DataContracts;

public class CreateCustomerModel {
    
        public required string FirstName { get; set; }
        
        public required string LastName { get; set; }
        
        public required string Email { get; set; }

        public required string PhoneNumber { get; set; }
        
        public required string Address { get; set; }
        
        public required string City { get; set; }
        
        public required string State { get; set; }
        
        public required string Zip { get; set; }
        
    }
    

File: WebShopUI/src/app/core/datacontracts/CreateCustomerModel.ts
export interface CreateCustomerModel {
    id: number
    firstName: string;
    lastName: string;
    email: string;
    phoneNumber: string;
    address: {
        street: string;
        city: string;
        state: string;
        zipCode: string;
    };
    dateOfBirth?: Date; // Optional field
    isActive?: boolean; // Optional field
}